src/test/java/uk/ac/vfb/geppetto/test/MultipleQueriesVFBQueryTest.java

```diff
@@ -280,22 +293,22 @@ public void testNANDedQueries() throws GeppettoInitializationException, Geppetto
 		rqEMF2.setBooleanOperator(BooleanOperator.NAND);
 		runnableQueriesEMF.add(rqEMF2);
 		
-		int count = aberDataSource.getNumberOfResults(runnableQueriesEMF);
+		int count = owleryDataSource.getNumberOfResults(runnableQueriesEMF);
 		try{
 			Assert.assertTrue(83<count);
 		}catch (AssertionError e) {
 			System.out.println("Fail: only " + count + " results returned, there should be 84+ results");
 			throw new AssertionError(e);
 		}
 			
-		QueryResults results = aberDataSource.execute(runnableQueriesEMF);
+		QueryResults results = owleryDataSource.execute(runnableQueriesEMF);
 
 		Assert.assertEquals("ID", results.getHeader().get(0));
 		Assert.assertEquals("Name", results.getHeader().get(1));
 		Assert.assertEquals("Definition", results.getHeader().get(2));
 		Assert.assertEquals("Type", results.getHeader().get(3));
 		Assert.assertEquals("Images", results.getHeader().get(4));
-		Assert.assertEquals(84, results.getResults().size());
+		Assert.assertTrue(results.getResults().size() > 80);
```

Comment:
@ddelpiano why more than 80 as opposed to checking the exact number? can the number vary?

Reply:
@tarelli The number of examples will always increase so as long as it's returning above an expected minimum them we can say it's working.